stream/src/main/java/org/apache/kafka/stream/processor/ProcessorProperties.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.clients.processor;
```

Comment:
@guozhangwang Do we need ProcessorProperties at all? Can we make all configs work the same way they do in the new producer and consumer which doesn't have separate *Properties classes?